File: OLSource1/v1/vs140/codesnippet/VisualBasic/how-to--modify-a-silverlight-screen-by-using-code_2.vb
        Private Sub Customers_SelectionChanged()
            FindControl("Customers_DeleteSelected").IsEnabled = True

            If Me.Customers.SelectedItem.CompanyName = "Great Lakes Food Market" Then

                FindControl("CompanyName1").IsVisible = False
                FindControl("Customers_DeleteSelected").IsEnabled = False

            End If

        End Sub

File: OLSource1/v1/vs140/codesnippet/CSharp/how-to--modify-a-silverlight-screen-by-using-code_2.cs
        partial void Customers_SelectionChanged()
        {
            this.FindControl("Customers_DeleteSelected").IsEnabled = true;

            if (this.Customers.SelectedItem.CompanyName == "Great Lakes Food Market")
            {
                this.FindControl("CompanyName1").IsVisible = false;
                this.FindControl("Customers_DeleteSelected").IsEnabled = false;
            }
        }